src/Audio/Properties/AssemblyInfo.cs

```diff
@@ -1,7 +0,0 @@
-﻿using System.Reflection;
```

Comment:
yes it does see [here](https://docs.microsoft.com/en-us/dotnet/api/system.reflection?view=netcore-1.1)